### PR TITLE
make the style of login

### PR DESCRIPTION
**Pull request by amanshawar**

### CODE REVIEW

File: client/src/components/Login/index.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+          <StyledLabel> Username* </StyledLabel>
+
+          <StyledInput type="text" name="username" placeholder="username" />
+
+          <StyledLabel> Password* </StyledLabel>
+
+          <StyledInput type="password" name="password" placeholder="password" />
+          <StyledP>
+            <StyledText>first time around here? </StyledText>
```
Comment (jema28) [inline client/src/components/Login/index.js]: Use a capital 'f' here.

File: client/src/components/Login/index.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    super(props);
+  }
+
+  render() {
+    return (
+      <React.Fragment>
+        <Title />
+        <Header title="Login to save your data!" />
+        <StyledForm>
+
```
Comment (jema28) [inline client/src/components/Login/index.js:33]: Why are these spaces here? On line 33, 35 etc?

File: client/src/components/Login/login.style.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+}
+`;
+export const StyledInput = styled.input`
+border:none;
+border-bottom:1px solid #7D7D7D;
+font-size: 20px;
+margin-bottom:16px;
+font-family:'Assistant';
+padding:7px 0px;
+@media (max-width: 650px) {
```
Comment (jema28) [inline client/src/components/Login/login.style.js]: Please indent the values in all styled components.

File: client/src/components/Login/login.style.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+`;
+export const StyledP = styled.div`
+display:flex;
+justify-content: center;
+flex-direction:row;
+`;
+
+export const StyledText = styled.div`
+font-size: 16px;
+font-family:'Assistant';
```
Comment (jema28) [inline client/src/components/Login/login.style.js]: You shouldn't have to repeat this, you can set the whole body font family to Assistant.

**Review by jema28 — CHANGES_REQUESTED** [review]
Small stylistic things but overall great job 👍

**Review by jema28 — APPROVED** [review]

**Review by orjwan98 — APPROVED** [review]

**Review by Hadeel-Salamin — APPROVED** [review]

**Pull request merged**